Source/MoreDotNet.Test/Extensions/Common/RandomExtensions/NextDoubleTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿namespace MoreDotNet.Tests.Extensions.Common.RandomExtensions
+{
+    using System;
+
+    using Moq;
+    using MoreDotNet.Extensions.Common;
+
+    using Xunit;
+
+    public class NextDoubleTests
+    {
+        private const int Counter = 1000;
+        private const double DoubleResult = 0.15;
+        private const double MinValue = -50;
+        private const double MaxValue = 150;
+
+        [Fact]
+        public void NextDouble_RandomIsNull_ShouldThrow_NullReferenceException()
+        {
+            Random random = null;
+
+            Assert.Throws<NullReferenceException>(() => random.NextDouble());
```

Comment:
You are testing the system implementation of NextDouble, no the actual overload extension method.